design/sketch.webidl

```diff
@@ -692,4 +696,58 @@ interface WebGPU {
 interface mixin WebGPUProvider {
     [Replaceable, SameObject] readonly attribute WebGPU gpu;
 };
+
 Window includes WebGPUProvider;
+
+// ****************************************************************************
+// DEBUGGING HELPERS
+// ****************************************************************************
+
+partial WebGPUProgrammablePassEncoder {
+    void pushDebugGroup(DOMString groupName);
+    void popDebugGroup(DOMString groupName);
+    void insertDebugMarker(DOMString markerName);
```

Comment:
The term "name" is used here, but "label" is used later; I think they should match (label seems good to me)

Reply:
Thanks @kainino0x!